engine/schema/src/com/cloud/storage/VMTemplateVO.java

```diff
@@ -146,6 +146,9 @@
     @Column(name = "dynamically_scalable")
     protected boolean dynamicallyScalable;
 
+    @Column(name = "direct_download")
+    private Boolean directDownload;
```

Comment:
Since it's `0` by default and cannot be `null`, it can be switched to a `boolean`